app/models/laptop.rb

```diff
@@ -0,0 +1,3 @@
+class Laptop < ActiveRecord::Base
+	years = ['2012', '2013', '2014']
```

Comment:
Wanted to create a dropdown for selecting the year in laptops. Deleting it for now